content/scripts/build.js

```diff
@@ -1163,23 +1263,34 @@ class Builder {
       renderedHtml = rawHtml;
     } else {
       let flaws;
-      [renderedHtml, flaws] = await this.renderMacros(
-        source,
-        rawHtml,
-        metadata
-      );
+      try {
```

Comment:
I don't understand the need for this `try...catch` block, since I would expect any error thrown from within `this.renderMacros()` would be caught and handled by the `try...catch` block around `this.processFolder()` one level up, right?

Reply:
I'll take another look. I've gone back and forth on these and perhaps the early versions are made redundant.